BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 Outernet Council Foundation
+# Copyright (c) Outernet Council and Contributors.
```

Comment:
My bad - I forgot to ask about this! The nmts repo uniformly uses this style and I accidentally copied the wrong version. Since there already is a contributors list here, maybe it makes sense to uniformly change to this version?